torch/fx/operator_schemas.py

```diff
@@ -335,7 +335,7 @@ def is_homogeneous_tuple(t):
 @compatibility(is_backward_compatible=False)
 def normalize_function(
     target: Callable,
-    args: tuple[Any],
+    args: tuple[Any, ...],
```

Comment:
@rec one nit I missed is this args and kwargs should probably use ParamSpec from the callable above

Reply:
@Skylion007 

Arg! 😬 

Right, of course, but this just got merged seconds ago.

I understand `ParamSpec` much better now, I shan't make this mistake in future, I hope.